cypress/integration/sidebar.spec.js

```diff
@@ -91,6 +92,29 @@ describe('Open the sidebar', function() {
 
 	})
 
+	it('Change image with sidebar open', function() {
+		cy.get('aside#app-sidebar').should('be.visible')
+
+		// check the sidebar is opened for the correct file
+		cy.get('aside#app-sidebar .app-sidebar-header .app-sidebar-header__title').should('contain', 'image1.jpg')
+
+		// open the next file (image2.png) using the arrow
```

Comment:
Please add screenshots between changes so we can easily follow the tests :)

Reply:
Haven't worked with Cypress before, still learning the ropes for it. :smile:

Reply:
You're doing great! :)